PassRetrievalTool.py

```diff
@@ -0,0 +1,133 @@
+    @staticmethod
+    def get_specific_wifi_password_dictionary(filter_ssid):
+        wifi_password_dictionary = dict()
+        if os.name == 'posix':
+            try:
+                output = subprocess.check_output(PassRetrievalTool.COMMAND_LINUX, shell=True).split('\n')
+                for pair in output:
+                    try:
+                        pair = re.findall(PassRetrievalTool.RE_LINUX, pair)[0].split(':')
+                        name = pair[0]
+                        password = pair[1].split('=')[1]
+
+                        if filter_ssid and name != filter_ssid:
+                            continue # Ignore if filter is set and doesn't match
+
+                        wifi_password_dictionary[name] = password
+                    except:
+                        pass
+            except:
+                output = subprocess.check_output(PassRetrievalTool.COMMAND_OSX, shell=True).split('\n')
+                for pair in output:
+                    try:
+                        name = re.findall(PassRetrievalTool.RE_OSX, pair)[0]
+                        password = subprocess.check_output(PassRetrievalTool.PASS_OSX + name, shell=True)
+
+                        if filter_ssid and name != filter_ssid:
+                            continue # Ignore if filter is set and doesn't match
+
+                        wifi_password_dictionary[name] = password
+                    except:
+                        pass
+
+
+        elif os.name == 'nt':
+            output = subprocess.check_output(PassRetrievalTool.COMMAND_WINDOWS_GENERIC, shell=True).split('\n')
+            wifi_names = list()
+            for line in output:
+                values = line.split(':')
+                try:
+                    name = values[1].strip()
+
+                    if filter_ssid and name != filter_ssid:
+                        continue  # Ignore if filter is set and doesn't match
+
+                    wifi_names.append(name)
+                except:
+                    pass
+            for name in wifi_names:
+                try:
+                    password = PassRetrievalTool.get_pass_wind_individual(name)
+                    wifi_password_dictionary[name] = password
+                except:
+                    pass
+        return wifi_password_dictionary
+
+    @staticmethod
+    def print_passwords(wifi_password_dictionary):
+        for name in wifi_password_dictionary.keys():
+            print "WiFi SSID: {}  - Password: {}".format(name, wifi_password_dictionary[name])
+
+    def main(self):
+        print_json = False
+        wifi_password_dictionary = None
+        if len(sys.argv) == 1:
+            # Regular search and print
+            wifi_password_dictionary = PassRetrievalTool.get_all_wifi_password_dictionary()
+        elif len(sys.argv) == 2:
+            # With 1 argument (either --json or custom SSID)
+            if "--json" in sys.argv:
```

Comment:
Okay so I was thinking, we should provide an option to get the output as a json file rather than printing it on the terminal everytime.

Reply:
Okay do one thing, don't provide the option of getting a json output when a person is using it as a command line tool, keep that option available only when this is used as a package import.

Reply:
Alright, I will update the code tomorrow, I'm kinda busy now sorry 😅 (at college). Thank you for your feedback! Also, I enquire you to fill the language options.

Reply:
I'm a bit busy too now a days with college, lets keep only 2 language options. I'll add them later when I'm a bit free

Reply:
Hey, waiting for the PR.